.travis.yml

```diff
@@ -1,15 +1,18 @@
-language: scala
+language: java
 sudo: false
 jdk:
   - oraclejdk8
-script:
-  - sbt +test
+env:
+  matrix:
+    - SCRIPT="mvn test"
+    - SCRIPT="sbt test"
 cache:
   directories:
-    - $HOME/.ivy2/cache
```

Comment:
I guess we should keep `~/.ivy2/cache` in the Travis cache now that it's a matrix build.